### ISSUE TITLE
make python-install requries sudo

### ISSUE THREAD

**Issue by stevekuznetsov** [krishauser/Klampt#12]
The `make python-install` requires `sudo`. This is incompatible with systems that do not have `sudo` (i.e. Docker containers) and I believe it's accepted as best practice not to call `sudo` inside `make`.
```
$ make python-install
[ 97%] Built target Klampt
[100%] Generating CMakeFiles/python_install_timestamp
/bin/sh: 1: sudo: not found
Python/CMakeFiles/python-install.dir/build.make:51: recipe for target 'Python/CMakeFiles/python_install_timestamp' failed
make[3]: *** [Python/CMakeFiles/python_install_timestamp] Error 127
CMakeFiles/Makefile2:1032: recipe for target 'Python/CMakeFiles/python-install.dir/all' failed
make[2]: *** [Python/CMakeFiles/python-install.dir/all] Error 2
CMakeFiles/Makefile2:1040: recipe for target 'Python/CMakeFiles/python-install.dir/rule' failed
make[1]: *** [Python/CMakeFiles/python-install.dir/rule] Error 2
Makefile:470: recipe for target 'python-install' failed
make: *** [python-install] Error 2
```

**Comment by krishauser** [comment]
Hi Steve,

This should be fixed now.

Cheers,
KH

**Comment by stevekuznetsov** [comment]
Fixed by 25e8b4e71d1e742451075938835b927bcdfd63d5
